pkg/util/mount/mount_linux_test.go

```diff
@@ -1551,6 +1617,17 @@ func TestFindExistingPrefix(t *testing.T) {
 			[]string{"test", "directory"},
 			false,
 		},
+		{
+			"with-fifo-in-middle",
+			func(base string) error {
+				testFifo := filepath.Join(base, "mount_test.fifo")
```

Comment:
cleanup?

Reply:
the test removes entire base dir on exit.